### DISCUSSION TITLE
How to Implement SD3 into Fooocus UI

### DISCUSSION THREAD

**Discussion by philparv** [lllyasviel/Fooocus#3159] [Q&A] [answered]
Hi, with the release of SD3, I would like to download the model from HF and then add it to my dropdown from the model settings, so I am able to switch between juggernaughtXL, SDXL, and SD3. I understand how to add it locally and run, the challenge is I am using a colab notebook to run with a faster GPU than my local machine, so trying to figure out how/where I should store the model/weights. Thank you in advance!

**Comment by mashb1t** [comment, marked as answer]
=> Fooocus does not yet support SD3, it's quite a journey ahead of us. See https://github.com/lllyasviel/Fooocus/discussions/3145#discussioncomment-9773154
